drivers/harfbuzz/SCsub

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+Import('env')
+
+env_hb = env.Clone()
+
+if (env['builtin_hb'] != 'no'):
+	# Thirdparty source files
+	thirdparty_dir = "#thirdparty/harfbuzz/"
+
+	thirdparty_sources = [
+		"src/hb-blob.cc",
+		"src/hb-buffer-serialize.cc",
+		"src/hb-buffer.cc",
+		"src/hb-common.cc",
+		"src/hb-face.cc",
+		"src/hb-font.cc",
+		"src/hb-ot-tag.cc",
+		"src/hb-ot-font.cc",
+		"src/hb-ot-layout.cc",
+		"src/hb-ot-map.cc",
+		"src/hb-ot-math.cc",
+		"src/hb-ot-shape.cc",
+		"src/hb-ot-shape-complex-arabic.cc",
+		"src/hb-ot-shape-complex-default.cc",
+		"src/hb-ot-shape-complex-hangul.cc",
+		"src/hb-ot-shape-complex-hebrew.cc",
+		"src/hb-ot-shape-complex-indic.cc",
+		"src/hb-ot-shape-complex-indic-table.cc",
+		"src/hb-ot-shape-complex-myanmar.cc",
+		"src/hb-ot-shape-complex-thai.cc",
+		"src/hb-ot-shape-complex-tibetan.cc",
+		"src/hb-ot-shape-complex-use.cc",
+		"src/hb-ot-shape-complex-use-table.cc",
+		"src/hb-ot-shape-normalize.cc",
+		"src/hb-ot-shape-fallback.cc",
+		"src/hb-ot-var.cc",
+		"src/hb-set.cc",
+		"src/hb-shape.cc",
+		"src/hb-shape-plan.cc",
+		"src/hb-shaper.cc",
+		"src/hb-ucdn.cc",
+		"src/hb-ucdn/ucdn.c",
```

Comment:
You only need one Unicode functions provider, since you already use ICU you might want to drop UCDN (you might also need to set `HAVE_ICU_BUILTIN` since you are not building it as a separate library).